lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -377,3 +415,30 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	skip: true,
+	ruleName,
+	config: [true],
+	customSyntax: 'postcss-css-in-js',
+	fix: true,
+
+	accept: [
+		{
+			code: "import styled from 'react-emotion'\nexport default styled.div` color: pink; `;",
+		},
+	],
+	reject: [
+		{
+			code: "import styled from 'styled-components';\nexport default styled.div` color: pink; color: orange; `;",
+			fixed:
+				"import styled from 'styled-components';\nexport default styled.div` color: orange; `;",
+			message: messages.rejected('color'),
+		},
+		{
+			code: "import styled from 'react-emotion'\nexport default styled.div` color: pink; color: orange; `;",
+			fixed: "import styled from 'react-emotion'\nexport default styled.div` color: orange; `;",
+			message: messages.rejected('color'),
+		},
+	],
+});
```

Comment:
[suggest] This code block using `testRule` is unnecessary because `skip: true` is set. Can you please remove it?

```suggestion
```

Reply:
Why keeping tests that have `skip: true` btw?

Reply:
We've been discussing deprecating the `postcss-css-in-js` package (see https://github.com/stylelint/postcss-css-in-js/issues/225).

Of course, we can remove the skipped cases (including other rules) soon, but it seems reasonable to do it with one PR.

```console
$ git grep postcss-css-in-js lib/rules | wc -l
      28
```

Also, we can do it after we determine to deprecate the package and remove it entirely from our codebase. It seems unnecessary to me to hurry.